Makefile

```diff
@@ -102,14 +103,15 @@ $(DEPLOY_CONTAINER_MARKER): Dockerfile build-containerized fetch-cni-bins
 	touch $@
 
 .PHONY: fetch-cni-bins
-fetch-cni-bins:
+fetch-cni-bins: dist/flannel dist/loopback dist/host-local
+
+dist/flannel dist/loopback dist/host-local:
 	mkdir -p dist
 	mkdir -p tmp-cni
```

Comment:
I don't think you need this temp dir. Just extract the files to the right place

`tar -xzf cni-amd64-v0.4.0.tgz -C dist ./flannel ./loopback ./host-local` (but using curl of course)